client/apps/edit2/components/admin/components/featuring/mentioned_list.jsx

```diff
@@ -0,0 +1,153 @@
+import colors from '@artsy/reaction/dist/Assets/Colors'
+import { connect } from 'react-redux'
+import { pluck } from 'underscore'
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { onAddFeature, setMentioned } from 'client/actions/editActions2.js'
+import * as ArticleUtils from 'client/models/article.js'
+import Artists from 'client/collections/artists.coffee'
+import Artworks from 'client/collections/artworks.coffee'
+import { ListItem } from 'client/components/autocomplete2/list'
+
+export class MentionedList extends Component {
+  static propTypes = {
+    article: PropTypes.object,
+    featured: PropTypes.object,
+    mentioned: PropTypes.object,
+    model: PropTypes.string,
+    onAddFeatureAction: PropTypes.func,
+    setMentionedAction: PropTypes.func
+  }
+
+  componentWillMount = () => {
+    const { model } = this.props
+
+    if (model === 'artist') {
+      this.getMentionedArtists()
+    } else {
+      this.getMentionedArtworks()
+    }
+  }
+
+  getMentionedArtists = () => {
+    const { article, setMentionedAction } = this.props
+    const artists = new Artists()
+    const ids = ArticleUtils.getMentionedArtistSlugs(article)
```

Comment:
Cool 🆒 Cool 🆒

Reply:
Didn’t get around to it, but we could add metaphysics/gravity support For fetching by [:id] and eliminate the backbone models at some point.